src/Widgets/DataPersonCardWidget/DataPersonCardWidgetComponent.tsx

```diff
@@ -66,8 +67,8 @@ const PersonCard = connect(function PersonCard({
               <tbody>
                 {email ? (
                   <tr>
-                    <th className="align-top">
-                      <i className="bi bi-envelope"></i>
+                    <th className="align-top" aria-label={getLocalizedEmail()}>
```

Comment:
* [ ] FYI This looks more like a workaround than an actual solution. 

The complaint is:

> Empty table header
> A <th> (table header) contains no text.

The question is: should we present the email address as a table from a semantic point of view?

From https://www.w3.org/WAI/tutorials/tables/tips/:

> Tables for Layout: Tables should not be used for layout purposes. Use Cascading Style Sheets (CSS) for layout. If there are already layout tables present, don’t use structural elements (like <th> or <caption>) and attributes discussed in this tutorial, and do add role="presentation" to the <table> element.